docs/asyncapi-document.md

```diff
@@ -84,4 +84,5 @@ In the sample code snippet below, notice how you can access the contents of the
   const channels = asyncapi.channels();
 ```
 
-> To learn about the various instances you can use to access the documents' content, look at the API of the AsyncAPI JavaScript Parser and the structure of [AsyncAPIDocument](https://github.com/asyncapi/parser-js/blob/master/API.md#module_@asyncapi/parser+AsyncAPIDocument)
\ No newline at end of file
+> To learn about the various instances you can use to access the documents' content, look at the API of the AsyncAPI JavaScript Parser and the structure of [AsyncAPIDocument](https://github.com/asyncapi/parser-js/blob/master/API.md#module_@asyncapi/parser+AsyncAPIDocument)
+>>>>>>> 0bc5e814b0cbb9da9665c694f778f7228b9ae00b
```

Comment:
```suggestion

```

@pratik2315 it's not clear if you addressed this feedback from Lukasz where he noticed this commit info left over in the doc